dotcom-rendering/docs/testing.md

```diff
@@ -59,50 +65,42 @@ We will **not** be writing snaphot test Components for the following reasons:
 -   Increased risk of false negatives and false positives - If the tests fail when there are no bugs, that is a false negative. If the tests pass when there are bugs present, that is a false positive.
 -   Developer time required to check snapshot test failures when simple non-breaking changes introduced, plus developer time required to review snapshot output in Pull Requests.
 
-## Cypress Tests
+## Playwright tests
 
-Cypress offers a solution for integration tests where tests are executed in a headless browser, using the actual browser apis. By executing at this level it provides an extremely realistic representation of a user interacting with the page.
+[Playwright](https://playwright.dev/) offers a solution for integration tests where tests are executed in a real browser. By executing at this level it provides a realistic representation of a user interacting with the page.
 
-### Types of Cypress tests
+### How to run locally
```

Comment:
We previously switched to run e2e tests [against the PROD](https://github.com/guardian/dotcom-rendering/pull/3698) build for consistency with CI and to reduce flakiness encountered with the dev build.

Now we've migrated to Playwright I want to try to remove this so the tests will automatically run against the DEV build locally and the PROD build in CI. This will will allow hot reloading for the test code _and_ the DCR code. I believe there shouldn't be any functional difference but I'll do this as a follow up ticket.